src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -31,6 +49,17 @@ public String todayDate(DateTimePart datePart) {
      *                   - 3-й элемент массива - день (число);
      */
     public Optional<LocalDate> getDate(Integer[] dateParams) {
+        try {
+            if (dateParams.length != 0) {
```

Comment:
do we need this checking?

Reply:
Yes, because we have test called getDateEmpty and we have to check whether our input data is empty or not.